assets/scss/_component-examples.scss

```diff
@@ -118,6 +118,7 @@
   position: relative;
   padding: 1rem;
   margin: 1rem (-$grid-gutter-width / 2);
+  overflow: scroll;
```

Comment:
`overflow: auto` is a better option here—it'll only show a scrollbar when necessary vs always showing one.

Reply:
DOH! good catch, feel stupid for using `scroll`. my CSS must be rusty. changed.